packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -492,6 +493,7 @@ function Preview({
                 }}
                 className="phone-screen"
               />
+              <RenderOutlinesOverlay />
```

Comment:
do we want to conditionally render this? does it make sense?

Reply:
sure, we can have that logic inside the component impl

Reply:
Done.